TiKiTuTo/Controller/InputHandler.cs

```diff
@@ -128,6 +129,37 @@ public bool GetApproval()
 
         }
 
+        public bool GetApprovalForExcelImport()
```

Comment:
If we add a string argument to getapproval(string prompt), we can use the same method in both cases (for timer and for excel)